Cargo.toml

```diff
@@ -9,7 +9,6 @@ repository = "https://github.com/rust-av/v_frame"
 
 [features]
 serialize = ["serde", "aligned-vec/serde"]
-wasm = ["wasm-bindgen"]
```

Comment:
Might be a good idea to leave it as no-op to stay compatible with rav1e

Reply:
I don't mind, but we have some [breaking changes](https://github.com/rust-av/v_frame/commit/0244b294b479026e55a784ac12284fa57f96bc9b) in the next release anyways, and it might be a good time to remove this immediately?

If you do mind, I'll just do a no-op wasm feature with a comment

Reply:
Let's land as-is then.